Formula/jack.rb

```diff
@@ -45,6 +45,13 @@ def plist; <<~EOS
     <dict>
       <key>Label</key>
       <string>#{plist_name}</string>
+      <key>EnvironmentVariables</key>
+      <dict>
+           <key>TMPDIR</key>
+           <string>/tmp</string>
+           <key>PATH</key>
+           <string>#{opt_bin}</string>
```

Comment:
This isn't just appending or prepending to PATH, so it should set something like
```
<string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```